moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js

```diff
@@ -201,12 +201,12 @@ export const ProjectActivityLogTableMaps = {
   moped_proj_partners: {
     label: "Partner",
     fields: {
-      added_by: {
+      created_by_user_id: {
         icon: "",
         label: "added By",
         type: "int4",
       },
-      date_added: {
+      created_at: {
         icon: "",
```

Comment:
Instead of renaming these entries, I believe we need to add the two new entries, so we can still display older activities that exist before this change. 

This is making me realize that I don't think I added the new fields on the PRs I worked on, I'll go back and check.

Reply:
reverted and added!